Makefile

```diff
@@ -171,7 +171,9 @@ receptorctl-test-venv/bin/pytest:
 	receptorctl-test-venv/bin/pip install -r receptorctl/test-requirements.txt
 
 receptorctl-tests: receptor receptorctl-test-venv/bin/pytest
-	cd receptorctl && ../receptorctl-test-venv/bin/pytest tests/tests.py
+	cd receptorctl && \
+		PATH=$(PATH):$(PWD) \
+			../receptorctl-test-venv/bin/pytest tests/
```

Comment:
Hmm, I thought this would make more sense this way, cause the `175` and `176` are the same command.
I'll update and change this and leave `176` at the same level as `175`, but I believe that they should not be at the same level as `174`, cause it will look like different commands.

The result will be:

```
cd receptorctl && ../receptorctl-test-venv/bin/pytest tests/tests.py
cd receptorctl && \
	PATH=$(PATH):$(PWD) \
	../receptorctl-test-venv/bin/pytest tests/
```